cluster/mesos/docker/docker-compose.yml

```diff
@@ -85,6 +85,7 @@ apiserver:
     --service-node-port-range=30000-32767
     --cloud-provider=mesos
     --cloud-config=/opt/mesos-cloud.conf
+    --allow-privileged
```

Comment:
This is default in upstream now. There are even e2e tests for this (which are fixed in this PR).